docs/quickstarts/sso-quickstart-customize.md

```diff
@@ -0,0 +1,642 @@
+
+5. If a dialog window appears to request permissions on behalf of the add-in, this means that SSO is not supported for your scenario and the add-in has instead fallen back to an alternate method of user authentication. This may occur when the tenant administrator hasn't granted consent for the add-in to access Microsoft Graph, or when the user isn't signed into Office with a valid Microsoft Account or Office 365 ("Work or School") account. Choose the **Accept** button in the dialog window to continue.
+
+    ![Permissions request dialog](../images/sso-permissions-request.png)
+
+    > [!NOTE]
+    > After a user accepts this permissions request, they won't be prompted again in the future.
+
+6. The add-in reads data from the signed-in user's OneDrive for Business and writes the names of the top 10 files and folders to the document. The following image shows an example of file and folder names written to an Excel worksheet.
+
+    ![OneDrive for Business information in Excel worksheet](../images/sso-onedrive-info-excel.png)
+
+### Outlook
+
+Complete the following steps to try out an Outlook add-in.
+
+1. In the root folder of the project, run the following command to build the project and start the local web server.
+
+    > [!NOTE]
+    > Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+2. Follow the instructions in [Sideload Outlook add-ins for testing](/outlook/add-ins/sideload-outlook-add-ins-for-testing) to sideload the add-in in Outlook. Make sure that you're signed in to Outlook with a user that's a member of the same Office 365 organization as the Office 365 administrator account that you used to connect to Azure while [configuring SSO](sso-quickstart.md#configure-sso) for the app. Doing so establishes the appropriate conditions for SSO to succeed. 
+
+3. In Outlook, compose a new message.
+
+4. In the message compose window, choose the **Show Taskpane** button in the ribbon to open the add-in task pane.
+
+    ![Outlook add-in button](../images/outlook-sso-ribbon-button.png)
+
+5. At the bottom of the task pane, choose the **Read my OneDrive for Business** button to initiate the SSO process. 
+
+6. If a dialog window appears to request permissions on behalf of the add-in, this means that SSO is not supported for your scenario and the add-in has instead fallen back to an alternate method of user authentication. This may occur when the tenant administrator hasn't granted consent for the add-in to access Microsoft Graph, or when the user isn't signed into Office with a valid Microsoft Account or Office 365 ("Work or School") account. Choose the **Accept** button in the dialog window to continue.
```

Comment:
Here, too, this is never going to happen in the context of this article, is it?

Reply:
(same response as indicated previously)